src/utils/shared/urls/index.ts

```diff
@@ -144,7 +144,7 @@ export const getIntlUrls = (
     politiciansHomepage: (filters?: Partial<{ state: string }>) => {
       const params = new URLSearchParams(filters).toString()
 
-      return `${countryPrefix}/politicians${params ? `?${params}` : ''}`
+      return `${countryPrefix}/politicians${params ? `?${params}#table` : ''}`
```

Comment:
This doesn't look right, you can add a `hash` arg and just use it

```suggestion
    politiciansHomepage: ({ filters, hash }: { filters?: Partial<{ state: string }>, hash?: string } = {}) => {
      const params = new URLSearchParams(filters).toString()

      return `${countryPrefix}/politicians${params ? `?${params}` : ''}${hash ? `#${hash}` : ''}`
```